app/models/changeset.rb

```diff
@@ -16,8 +16,14 @@ def self.find(repo, previous_commit, commit)
     end
 
     new(comparison, repo, previous_commit, commit)
-  rescue Octokit::NotFound, Octokit::InternalServerError
-    new(NullComparison, repo, previous_commit, commit)
+  rescue Octokit::NotFound, Octokit::InternalServerError => e
+    error = case e
```

Comment:
I try to avoid creating local variables with the same name as methods. Maybe `errmsg`?